CHANGELOG.md

```diff
@@ -15,6 +15,11 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#2249](https://github.com/realm/SwiftLint/issues/2249)
 
+* Add new rule `vertical_whitespace_between_cases` to warn against crowded
```

Comment:
I'd suggest to mention this is a opt-in rule.

Reply:
Done.